app/views/schools/setup_school_cohort/expect_any_ects.html.erb

```diff
@@ -1,27 +1,33 @@
-<% title = "Does your school expect any ECTs in the next academic year?" %>
+<% title = "Does your school expect any new ECTs in the 2022 to 2023 academic year?" %>
```

Comment:
```suggestion
<% title = "Does your school expect any new ECTs in the 2022 to 2023 academic year?" %>
```

These dates should not be hard coded

Reply:
Yeah thats what i said in convo to you - if someone can change it that would be great - its out of my remit.